content/versions-comparison/3.0-news.json

```
@@ -1,27 +1,27 @@
 [
   {
-    "title":"Product-related enriched entities: the Reference entities",
+    "title":"Reference entities to manage additional product data",
```

```suggestion
    "title":"Reference entities to manage product-related data",
```
Sounds better and is shorter. What do you think?